contrib/kubernetes/README.md

```diff
@@ -0,0 +1,44 @@
+# Kubernetes  Setup for DataHub
+
+## Introduction
+This directory provides the Kubernetes setup for DataHub.  This is the first version with simple YAML files.  
+The next version will contain DataHub [Helm](https://helm.sh/) chart that can be published to [Helm Hub](https://hub.helm.sh/)
```

Comment:
This is exciting!
I'm definitely no expert in Kubernetes world and might ask some noob questions below :)

1. Is `Helm Hub` the counterpart of `Docker Hub` in Kubernetes world?
2. Will we have an Helm chart for each container? I mean one for `datahub-gms`, one for `datahub-frontend` etc. just like Docker. Or, will we have a single `DataHub` chart which contains multiple containers?
3. Docker Hub has a very nice integration with Github and I've setup [automated builds](https://docs.docker.com/docker-hub/builds/) on Docker Hub .Each new commit gets built on Docker Hub and automatically triggers a new release of all DataHub Docker images. Will we have a similar integration with Helm charts?

Reply:
1. Helm3 Supports OCI specification and can be pushed to all Container Registries.  But this is an experimental feature as of now.  Helm Hub is supported as of now.  The support for docker-registry can be tracked with [issue here](https://github.com/helm/helm/issues/5861)
2. We will have one Helm chart for datahub.  Helm packaging of Kubernetes is more like "RPM/APT" packaging in linux world.  One package which deploys all the components of Datahub.  You can look at airflow Helm chart [here](https://github.com/bitnami/charts/tree/master/bitnami/airflow)
3. I'll explore [Custom build hooks](https://docs.docker.com/docker-hub/builds/advanced/#custom-build-phase-hooks) and see if it is possible.  It is a matter of running `helm chart push` command.  Also, pushing it to [ghpages](https://medium.com/@kwonghung.yip/hosting-helm-repository-on-github-pages-and-using-travis-ci-pipeline-to-build-the-chart-e1a8ad4a1e1c)  is very interesting

Reply:
Sounds good, would be perfect if we'd be able to auto-build latest chart with every commit.